1.table.sql

```diff
@@ -0,0 +1,42 @@
+use CostumeDB
+go
+drop table if exists Sales
+go
+
+create table dbo.Sales(
+    CustomerId int not null identity primary key,
+    CustomerName varchar(50) not null 
+        constraint ck_customer_name_cannot_be_blank check(CustomerName <> ''), 
+    CostumesBought varchar(25) not null
+        constraint ck_costume_bought_cannot_be_blank_and_is_specific check(CostumesBought in ('American Girl Doll', 'Artist', 'Bumble Bee', 'Colonial Boy','Colonial Girl','Elephant','Fire Man','Police Man', 'Princess', 'Zebra')),
+    CostumeSize varchar(2) not null 
+        constraint ck_costume_size_cannot_be_blank_and_is_specific check(CostumeSize in ('XS', 'S', 'M', 'L', 'XL')),
+    Quantity int not null 
+        constraint ck_quantity_must_be_at_least_1 check(Quantity > 0),
+    PriceSoldPerCostume int not null 
+        constraint ck_price_sold_per_costume_there_are_fixed_prices check(PriceSoldPerCostume in (15,17,20,22,25,27,30)),
+    DateBought date not null,
```

Comment:
AS: -2 DateBought must be after the bussiness opened.

Reply:
I have made the amendments.  On the spec it mentioned that she was supplying fam and friends before the business opened - is that not a reason not to have the date bought constraint after the business opened?

Reply:
I would assume that would not be included in the db since that was before it becase an official business.